sdks/python/apache_beam/transforms/display_test.py

```diff
@@ -114,6 +115,33 @@ def display_data(self):
     with self.assertRaises(ValueError):
       DisplayData.create_from_options(MyDisplayComponent())
 
+  def test_vp_display_data(self):
+    class TestOptions(PipelineOptions):
+      @classmethod
+      def _add_argparse_args(cls, parser):
+        parser.add_argument(
+            '--int_flag',
+            type=static_value_provider_of(int),
+            help='int_flag description')
+        parser.add_argument(
+            '--str_flag',
+            type=static_value_provider_of(str),
+            help='str_flag description')
+    options = TestOptions(['--int_flag', '1', '--str_flag', '/dev/null'])
+    # TODO: Make flags be capable of having
```

Comment:
What does this TODO mean? Also could you write it in TODO(jira issue) or TODO(user name)?